### PR TITLE
Search by URI must return result

### PR DESCRIPTION
**Pull request by gabrielfs7**
Search by URI must return result

https://oat-sa.atlassian.net/browse/ADF-268

### CODE REVIEW

File: models/classes/search/SearchProxy.php [hunk 1]
```diff
@@ -143,13 +143,13 @@ private function executeSearch(SearchQuery $query): ResultSet
             return new ResultSet([], 0);
         }
 
-        if ($this->isForcingDefaultSearch($query) || !$this->getAdvancedSearchChecker()->isEnabled()) {
-            $result = $this->getIdentifierSearcher()->search($query);
+        $result = $this->getIdentifierSearcher()->search($query);
```
Comment (gabrielfs7) [inline models/classes/search/SearchProxy.php:146]: - [ ] I still need to check if only this search is used by UDIR

**Review by boajer — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-advanced-criteria.playground.kitchen.it.taocloud.org:42161/

**Pull request merged**